validator/src/validation/cards.rs

```diff
@@ -0,0 +1,29 @@
+use validation::Validator;
+use card_validate::Validate;
+
+
+pub fn validate_credit_card(card: &str) -> bool {
+    println!("{:?}", card);
```

Comment:
you can remove that

Reply:
Woups, had a test failing and forgot to remove this line, sorry...